ViteMaDose/Views/Home/Cells/HomeTitleCell.swift

```diff
@@ -87,7 +87,7 @@ extension HomeTitleCell {
 
     static var lastStatsAttributedText: NSMutableAttributedString {
         let titleFont: UIFont = .rounded(ofSize: 26, weight: .bold)
-        let titleText = "Dernières statistiques"
+        let titleText = Localization.Home.last_stats
```

Comment:
Same remark here (use `Localization.Home.last_stats` two lines below instead of `titleText`)